docs/models_list.rst

```diff
@@ -158,6 +144,27 @@ Below is an example of the ``default`` data type for each filter:
 
   View the `complete list of Linz filters <https://github.com/linzjs/linz/tree/master/lib/formtools/filters>`_.
 
+list.groupActions
+=================
+
+``list.groupActions`` can be used to define certain actions that are only available once a subset of data has been chosen.
+
+Each record displayed on a model index has a checkbox, checking two or more records creates a group. If ``groupActions`` have been defined for that model, those actions will become chooseable by the user.
```

Comment:
I think it is `choosable`, both spellings look strange to me though! :)

Reply:
Fixed :)